lib/elixir/test/elixir/macro_test.exs

```diff
@@ -355,8 +355,8 @@ defmodule MacroTest do
   end
 
   test "range to string" do
-    assert Macro.to_string(quote do: (1 .. 2)) == "1..2"
     assert Macro.to_string(quote do: unquote(-1 .. +2)) == "-1..2"
+    assert Macro.to_string(quote do: Foo.integer..3) == "Foo.integer()..3"
```

Comment:
@josevalim the idea to have this test is to be sure that we're calling `to_string` on arguments and not `inspect` for example, since in `quote do: (1 .. 2)` range arguments are just literals.